aws_db_instance.tf

```diff
@@ -3,47 +3,53 @@
 #}
 
 resource "aws_db_instance" "secureInstance" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
-  engine               = "mysql"
-  engine_version       = "5.7"
-  instance_class       = "db.t2.micro"
-  name                 = "mydb"
-  backup_retention_period = 90
+  allocated_storage                   = 20
+  storage_type                        = "gp2"
+  engine                              = "mysql"
+  engine_version                      = "5.7"
+  instance_class                      = "db.m5.xlarge"
+  name                                = "mydb"
+  backup_retention_period             = 90
   iam_database_authentication_enabled = true
-  auto_minor_version_upgrade = true
+  auto_minor_version_upgrade          = true
+  username                            = "<some_random_username>"
 }
 
 resource "aws_db_instance" "rdsBackupDisbaled" {
-  allocated_storage    = 20
-  storage_type         = "gp2"
-  engine               = "mysql"
-  engine_version       = "5.7"
-  instance_class       = "db.t2.micro"
-  name                 = "mydb"
-  backup_retention_period = 0
+  allocated_storage                   = 20
+  storage_type                        = "gp2"
+  engine                              = "mysql"
+  engine_version                      = "5.7"
+  instance_class                      = "db.m5.xlarge"
+  name                                = "mydb"
+  backup_retention_period             = 30
+  iam_database_authentication_enabled = true
+  username                            = "<some_random_username>"
 }
 
 resource "aws_db_instance" "rdsIamAuthDisabled" {
-  allocated_storage    = 20
```

Comment:
**Accurics remediation**
 
**[HIGH]: CA Certificate used for RDS is old**
 
**Vulnerability**
Ensure Certificate used in RDS instance is updated
 
**Remediation**
AWS rotates CA Certificates for RDS on a regular basis, ensure your RDS Instance is always updated to avoid connectivity issues.

source: accurics-